package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "chaos-recipe-overlay",
-  "version": "1.0.0",
+  "version": "1.1.0",
```

Comment:
There's no need for version bumping here, the version will change when the next release is drafted.

Reply:
Oh, didn't know that, thanks!